usagov/usagov-2021
# USAGOV-1603 editable labels

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1603

## Description

Adds content field for basic page that only display for certain page types:

- For homepage, fields for title and description used in the new call out linking to benefits search landing page
- For topic nav, fields for description used in the new call out linking to benefits search landing page
- For benefits landing page, title and descriptions used above benefit finder section and above the category search section

Doesn't make any visible changes, but updates twig templates to remove hard coded strings and use CMS fields instead.
 
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



When editing the home page, should see field group for "Homepage Benefits Callout" labels. This group only displays on home page. The values are translatable and should be different when editing English or Spanish translation.

![image](https://github.com/usagov/usagov-2021/assets/493854/8c949983-1db2-439b-b5ae-88e66b3e89ca)

When editing a navigation cards page, should see a field group for "Benefits Callout with just a description field.

![image](https://github.com/usagov/usagov-2021/assets/493854/396274e9-4fd4-40d1-ad84-d64ac4354543)

When editing a benefits category search page, should see a field group for "Benefits Search Landing Page". This group does not display on other page types.

![image](https://github.com/usagov/usagov-2021/assets/493854/75dc32fb-33a5-416a-aaf5-d137688184f9)

The Benefit Category Search group now only displays on Standard Pages. This group does not display on other page types.

![image](https://github.com/usagov/usagov-2021/assets/493854/d3e5f510-71ec-4de6-a1a6-fd1c52ffc740)

### Requires New Config

### Requires New Content
- [X] Yes - On the home page,  I made the call out fields required, but should they be optional and only render the call out if they have content?

### Validation Steps
Assuming you have set up the benefits search pages per `docs/BenefitSearch`. See the **Setup** section.

- Edit English & Spanish home page, confirm call out field group displays.
- Edit English & Spanish navigation cards confirm call out field group displays.
- Edit English & Spanish benefits landing page confirm the landing page field group displays.
- Confirm "Benefits Category Search" group only shows when editing a basic page.

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
